README.md

```diff
@@ -69,13 +69,24 @@ Example:
 
 A user who visits the proxy will be redirected to an OAuth login with OpenShift, and must grant
 access to the proxy to view their user info and request permissions for them. Once they have granted
-that right to the proxy, it will check whether the user has the required permissions. If they do 
+that right to the proxy, it will check whether the user has the required permissions. If they do
 not, they'll be given a permission denied error. If they are, they'll be logged in via a cookie.
 
 Run `oc explain subjectaccessreview` to see the schema for a review, including other fields.
 Specifying multiple rules via a JSON array (`[{...}, {...}]`) will require all permissions to
 be granted.
 
+##### Require specific permissions per host with `--openshift-sar-by-host=JSON`
+
+This is similar to the `--openshift-sar` option but instead of the rules applying to all hosts, you
+can set up specific rules that are checked for a particular upstream host. Using a JSON object the
+keys are hostnames and the value is a JSON array of SAR rules.
```

Comment:
How does this work alongside --openshift-sar? Will --openshift-sar= compound on each host-specific sar?

Reply:
If they work together in that way, it might then be better to just change --openshift-sar to allow in its array a combination of global rules and host-specific rules.

Reply:
Yes they are just compounded together. I was initially going down that path but I couldn't figure out the best way to distinguish between a single SAR rule object and an object mapping of hosts to SAR rule or array of SAR rules.

Reply:
OK, please add a mention that both options work together in the README.

Reply:
Done